maxim/test_runs/run_utils.py

```
@@ -14,39 +14,49 @@
+            )
+        elif column_type in ("VARIABLE", "NULLABLE_VARIABLE"):
+            # Skip nullable variables with None values to avoid invalid payloads
+            if column_type == "NULLABLE_VARIABLE" and row[column_name] is None:
+                continue
+            variables[column_name] = Variable(
+                type_="text", payload={"text": row[column_name]}
+            )
+    return variables
+
+
+def get_input_expected_output_and_context_from_row(
```

Ruff (0.11.9)

Missing return type annotation for public function `get_input_expected_output_and_context_from_row`

(ANN201)